app/components/Chatbot/widgit.css

```diff
@@ -69,8 +69,10 @@
 
 .widget-ask .send {
   position: absolute;
-  right: var(--spacing-8);
-  bottom: var(--spacing-8);
+  right: var(--spacing-16);
+  bottom: 24px;
+  z-index: 5;
```

Comment:
does z-index do anything for a component with position absolute? (see https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context)